appintro/src/main/java/com/github/paolorotolo/appintro/AppIntro2.kt

```diff
@@ -3,32 +3,32 @@ package com.github.paolorotolo.appintro
 import android.graphics.drawable.Drawable
 import android.os.Bundle
 import android.view.View
-import android.widget.FrameLayout
 import android.widget.ImageButton
-import android.widget.LinearLayout
 import androidx.annotation.ColorInt
+import androidx.constraintlayout.widget.ConstraintLayout
 
 abstract class AppIntro2 : AppIntroBase() {
 
-    var backgroundView: View? = null
+    var backgroundView: Int? = null
```

Comment:
Please rename to either:

```
@IdRes var backgroundResource: Int? = null
```

or 

```
var backgroundDrawable: Drawable? = null
```

Reply:
Here is the problem, setting it to introduce reduces customizability and setting it to drawable requires API level 16... So what am I supposed to do now?

Reply:
We can actually have both and wrap the `Drawable` version in a if-then-else API => 16

Reply:
Okay. Take a look at my new issue @cortinico